geometry_msgs/PoseStamped target_pose
string target_map
---
bool success
string message
---
string current_status
